### ISSUE TITLE
iOS Chrome createBrowserHistory.goBack BUG

### ISSUE THREAD

**Issue by Frenzzy** [rackt/history#67]
#### Initial data:
**Device:** iPhone 5
**OS:** iOS v9.0.1
**Browser:** [Chrome](https://itunes.apple.com/en/app/chrome-brauzer-ot-google/id535886823) v45.0.2454.89
**Bootstrap code:**
```js
import { createBrowserHistory } from 'history' // v1.11.0

const history = createBrowserHistory()
```

#### Steps to reproduce:
1. Go to www.google.com
2. Go to our site
3. Invoke `history.pushState(null, '/demo/path1')`
4. Invoke `history.pushState(null, '/demo/path2')`
5. Invoke `history.goBack()`

#### Expected result:
The history goto our site `/demo/path1`

#### Actual result:
The history goto `www.google.com`

P.S.: It is work fine in Safari.

**Comment by dzannotti** [comment]
Having the same issue in chrome desktop for osx too and safari

**Comment by mjackson** [comment]
Can you please create a failing test case for this? If you have two calls to `pushState` in the same tick (i.e. one right after the other), `goBack` may actually take you back to the previous site. I would not consider this a `history` bug, but rather just a quirk of how `pushState` works in Chrome.

**Comment by mjackson** [comment]
Would you please follow up here, @Frenzzy? Otherwise, please close. Thanks :)

**Comment by Frenzzy** [comment]
It seems this is Chromium bug: https://code.google.com/p/chromium/issues/detail?id=343062

**Comment by frenzzy** [comment]
It seems it was fixed in #146 and released at [Dec 6, 2015 (v1.14.0)](https://github.com/rackt/history/blob/master/CHANGES.md#v1140)
